jsonnet/jsonnet.bzl

```diff
@@ -47,194 +47,210 @@ _JSONNET_FILETYPE = [
     ".json",
 ]
 
+def _add_prefix_to_imports(label, imports):
+    imports_prefix = ""
+    if label.workspace_root:
+        imports_prefix += label.workspace_root + "/"
+    if label.package:
+        imports_prefix += label.package + "/"
+    return [imports_prefix + im for im in imports]
+
 def _setup_deps(deps):
-  """Collects source files and import flags of transitive dependencies.
-
-  Args:
-    deps: List of deps labels from ctx.attr.deps.
-
-  Returns:
-    Returns a struct containing the following fields:
-      transitive_sources: List of Files containing sources of transitive
-          dependencies
-      imports: List of Strings containing import flags set by transitive
-          dependency targets.
-  """
-  transitive_sources = depset(order="postorder")
-  imports = depset()
-  for dep in deps:
-    transitive_sources += dep.transitive_jsonnet_files
-    imports += ["%s/%s" % (dep.label.package, im) for im in dep.imports]
-
-  return struct(
-      transitive_sources = transitive_sources,
-      imports = imports)
+    """Collects source files and import flags of transitive dependencies.
```

Comment:
Please use 2-space indentation to conform with the existing convention in the file.

Reply:
Oh my, having taken a closer look, it seems the file is a mess of inconsistent spacing.  Based on https://github.com/bazelbuild/buildtools/issues/34 I think you're doing the right thing.

Would you mind splitting the buildifier formatting out into its own commit (and optionally it's own pull request) so that your actual changes are easier to digest?